client/src/components/PropertyDetails.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+import('env2')('./config.env');
+const key = process.env.Currency_API_KEY;
```

Comment:
I didn't spot this the first time around. Can you make this environment variable UPPER_SNAKE_CASE please?

Reply:
Done